ios_swift/ResumeARStarter/ViewController.swift

```diff
@@ -151,16 +151,29 @@ class ViewController: UIViewController, ARSCNViewDelegate {
                 return
         }
 
-        // Retrieve credentials
-        guard let vrApiKey = credentials["visualrecognitionApi_key"] as? String, !vrApiKey.isEmpty,
-            let url = credentials["cloudantUrl"] as? String, !url.isEmpty else {
+        // Retrieve Cloudant credentials
+        guard let url = credentials["cloudantUrl"] as? String, !url.isEmpty else {
                 return
         }
 
-        self.visualRecognition = VisualRecognition.init(apiKey: vrApiKey, version: self.VERSION)
+        // Set the Watson credentials for Visual Recognition service from the BMSCredentials.plist
+        // If using IAM authentication
+        if let apiKey = credentials["visualrecognitionApikey"] as? String {
+
+            // Create service sdks
+            self.visualRecognition = VisualRecognition(version: self.VERSION, apiKey: apiKey)
+
+        // Else for user/pwd authentication
```

Comment:
you write user/pwd here but on L172 the api key is still being passed in. IIRC VizRec didn't have a user/pwd combo, wasn't it just the key that changed? `visualrecognitionApi_key` vs `visualrecognitionApikey` ? if that's the case then make the comment correct.